examples/certification/certified-counter/src/backend/src/lib.rs

```diff
@@ -1,16 +1,15 @@
-use ic_cdk::export::candid::CandidType;
+use candid::CandidType;
```

Comment:
This example project was broken because of this change, which is part of the motivation for these changes